content/codespaces/getting-started-with-codespaces/getting-started-with-your-java-project-in-codespaces.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Getting started with your Java project in Codespaces
+shortTitle: Getting started with your Java project
+intro: 'You can create a custom dev container with all the tools necessary to get started with your Java project in {% data variables.product.prodname_codespaces %}.'
+product: '{% data reusables.gated-features.codespaces %}'
+versions:
+  free-pro-team: '*'
+topics:
+  - Codespaces
+---
+
+
+### Introduction
+
+This guide shows you how to set up your Java project in {% data variables.product.prodname_codespaces %}. It will take you through an example of opening your project in a codespace,and adding and modifying a dev container configuration from a template.
```

Comment:
```suggestion
This guide shows you how to set up your Java project in {% data variables.product.prodname_codespaces %}. It will take you through an example of opening your project in a codespace, and adding and modifying a dev container configuration from a template.
```